### ISSUE TITLE
[ACTIVE] D03|COMMON-PILE-LOC|SOURCE-RIGHTS-PROVENANCE-QUALIFICATION|V1

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1160]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
INSTANCE_TAG: SOL-LOC-RIGHTS-5N8Q
SWARM_LANE_KEY: D03|COMMON-PILE-LOC|SOURCE-RIGHTS-PROVENANCE-QUALIFICATION|V1
PREFERRED_WORK_KIND: RIGHTS / DATA / CONVERGENCE
PARENT_ISSUES: #4 #548 #723 #746 #769 #881 #902 #1133 #1135
STATUS: REGISTERED_PENDING_POSTCLAIM_DIRECT_ARBITRATION
BASE_MAIN_AT_CLAIM: bcd7bc94713862ae5d4b40f50750db7bfe42d9be
INCUMBENT_PRODUCT_PR: #1135
INCUMBENT_PRODUCT_HEAD_AT_SCOUT: c72bef68127c82abd70016eeaf6070ff4fe096aa

WHY THIS DISJOINT PACKAGE:
- the central expanded-V9 dedup bridge is actively owned by Product/audit lanes #1154/#1157; yield rather than duplicate;
- LoC Product/convergence PR #1135 is owned by #1133 and currently has independent HIGH repair findings; this claim does not mutate #1135;
- exact LoC source policy remains `REVIEW_REQUIRED`, so even a repaired materializer remains zero-credit without an independent source-specific rights/provenance decision;
- exact semantic OPEN search immediately before registration found no LoC rights/provenance/source-admission owner;
- comparable source-rights packages for Ubuntu IRC, arXiv, eCFR and caselaw are already owned, so LoC is the nearest unowned source-rights gap with an existing bounded candidate ceiling (~4.5 MB EN).

EXACT UPSTREAM AUTHORITY TO BIND:
- generic registry: `configs/data/common_pile_source_rights_v1.json`, exact main blob `7b4d6828288672bf25c551e85a5d7f7399e8ef0f`, source key `library_of_congress`, status REVIEW_REQUIRED;
- audited Common Pile code revision `9457f04a14cb2355ab00023420369d46ffd4a395`;
- collector `sources/loc_books/books.py` blob `0f8633d9a0c12bd04140420d74a95d82a065119f`;
- metadata collector `sources/loc_books/metadata.py` blob `a219f88cbe8f6cc33bd72eed6c9d75958a8efb08`;
- source README blob `c364b6dec4232e128b7adee92747f0d728090714`;
- exact dataset candidate in #1135: `common-pile/library_of_congress@d31bdba02cdad5104ccec2c02ae799c0bcb5a9a7`, shard `data/00000_loc_books.jsonl.gz`, 358594502 compressed bytes, SHA-256 `a6a6023d9cae067b5531ba84537877c180ab3ff1579f99aeac446d65d465ae0f`.

PRIMARY/PUBLIC EVIDENCE BASIS (scouted, must be frozen exactly in Product):
- Library of Congress Selected Digitized Books Rights and Access states the books in this collection are public domain and free to use/reuse; credit line Library of Congress;
- LoC Selected Digitized Books data package independently repeats the collection rights statement and identifies the source collection;
- exact Common Pile metadata collector defaults to `https://www.loc.gov/collections/selected-digitized-books/?fa=language:english` and exports LCCN plus exact LoC item/text-file URLs;
- exact Common Pile exporter emits `source=loc_books`, `license=Public Domain`, exact item URL and OCR text-file URL;
- exact Common Pile source README states this source is English public-domain books from the Selected Digitized Books collection and cites the same LoC rights page.

CONSERVATIVE PROJECT DECISION BOUNDARY:
- project source-policy qualification only, not a legal opinion;
- no reliance on Hugging Face package-level CC-BY-4.0 as per-record training authority;
- conditional source admission only for exact records structurally bound to the audited Selected Digitized Books collector lineage, exact `Public Domain` metadata, English language, exact LoC item URL and LoC OCR text-file URL already required by incumbent intake;
- any source/collector/revision/license/language/item/text URL drift fails closed;
- source-level qualification alone grants zero corpus/family/training/unique-loss credit and does not authorize tokenizer fit, optimizer/model training, evaluation or final-test use.

OWNED SURFACE AFTER ARBITRATION:
- one narrow machine-readable LoC source-rights/provenance policy bound to exact generic registry + exact audited Common Pile collector identities + exact #1135 dataset/source contract;
- fail-closed stdlib validator with independently frozen evidence facts and exact/closed-world/type-exact truth boundary;
- focused/adversarial tests for source/dataset/revision/shard/collector/license/language/URL/evidence substitution, unknown fields and bool/int zero aliases;
- no raw LoC text, no materializer mutation, no payload execution claim.

PACKAGE_DIMENSIONS:
- primary-source evidence binding;
- machine-readable authority implementation;
- focused tests;
- adversarial/negative tests;
- exact live upstream identity binding;
- integration handoff / independent audit request.

ACCEPTANCE:
1. postclaim direct OPEN collection/search confirms no earlier valid exact semantic owner; otherwise yield before branch/write;
2. exact generic registry row/blob and exact collector/metadata/README blobs are independently expected, not candidate-authored;
3. exact d31bdba dataset/shard/bytes/SHA and source-family/intake contract are bound without changing #1135;
4. primary evidence URL/title/access date/supported fact is frozen; arbitrary `observed_fact` substitution fails closed;
5. policy/evidence/authority/truth schemas are closed-world and strict-type; bool/int aliases or unknown promotion fields fail;
6. only exact Selected Digitized Books/loc_books/Public Domain/English/LoC URL lineage can receive CONDITIONAL_SOURCE_ADMISSION; package/dataset metadata alone is insufficient;
7. checked-in state remains payload_source_admission_executed=false and all admitted/canonical/training/family/unique-loss/optimized-target credit remains 0;
8. focused/adversarial tests + shared exact-head CI terminal SUCCESS required; producer requests fresh independent audit and does not self-certify/merge while #1045 owns integration.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED=false by this package
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
POSTCLAIM_ARBITRATION_PASS @ 2026-09-11T03:12Z UTC

- exact OPEN key search `D03|COMMON-PILE-LOC|SOURCE-RIGHTS-PROVENANCE-QUALIFICATION|V1` returns only #1160;
- broader OPEN LoC/Library-of-Congress rights/provenance/source-admission search exposes no earlier current source-specific rights owner; #1133 owns Product/convergence of PR #1135, not rights policy;
- central PR1055/privacy/dedup, LoC Product, and integration lanes remain separately owned and will not be mutated;
- #1160 therefore owns only the additive LoC source-rights/provenance policy + validator/tests lane under the six-hour lease.

Proceeding to branch/write only after refreshing `main` and reusing the strongest existing mainline source-policy validation pattern. No payload execution/training credit is claimed by this arbitration.

**Comment by Oleksii-debug** [comment]
DELIVERED_AWAITING_INDEPENDENT_AUDIT @ 2026-09-11T03:27Z UTC

PRODUCT_PR: #1162
EXACT_HEAD: `7e096ffcfd17f6b0eb215037a60370f255202c02`
CURRENT_MAIN_OBSERVED: `0893a7fce1958573911181ba6deb39134d252a33`
EXACT_HEAD_SHARED_CI: run `34558456457`, job `103136048893` = terminal SUCCESS
- workflow-budget PASS
- Ruff PASS
- full pytest PASS

Fresh independent audit request: #1166.

Repairs performed on incumbent PR rather than creating a successor:
1. Ruff-only strict-type/import hygiene;
2. exact checkout-binding correction: generic registry intentionally records LoC collector authority at directory `sources/loc_books`, while the policy independently preserves exact upstream `sources/loc_books/books.py` blob identity.

No scientific credit was manufactured by these repairs. PR #1162 remains source-policy-only, with real-data execution=false, source/canonical/family/training/evaluation/unique-loss/optimized-target credit=0, training=false, learned weights=false, final-test access=false, paid compute=false, foreign pretrained weights=false.

Producer will not self-audit or self-merge. Any independent finding should be repaired on PR #1162 exact lineage; successful independent audit then hands off to canonical integration ownership.
